java/quickstart/src/test/java/com/example/cloud/bigtable/quickstart/it/QuickstartIT.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.example.cloud.bigtable.quickstart.it;
+
+import static org.junit.Assert.assertTrue;
+
+import com.example.cloud.bigtable.quickstart.Quickstart;
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Cloud Bigtable Quickstart integration tests
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class QuickstartIT {
+  
+  private final String instanceId = System.getProperty("bigtable.test.instance");
+  private final String tableId = formatForTest("my-table");
+  private final String columnFamilyName = "my-column-family";
+  private final String columnName = "my-column";
+  private final String data = "my-data";
+
+  private final String projectId = System.getenv("GOOGLE_CLOUD_PROJECT");
```

Comment:
Does CBT need to support this?

Reply:
No, hence not part of running the quickstart instructions, but only the test.
This just keeps it in sync with tests in [java-docs-samples](https://github.com/GoogleCloudPlatform/java-docs-samples) in how we set the project for all our tests on Kokoro.